proto/osmosis/gamm/pool-models/concentrated-liquidity/concentratedPool.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+// this is a legacy package that requires additional migration logic
+// in order to use the correct packge. Decision made to use legacy package path
+// until clear steps for migration logic and the unknowns for state breaking are
+// investigated for changing proto package.
+package osmosis.gamm.poolmodels.concentratedkiquidity.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v12/x/gamm/pool-models/concentrated-liquidity";
+
+message Pool {
```

Comment:
nit: proto doc

Reply:
We don't have anything in the Pool yet + things can change, will leave empty for now, but we'll def make sure to come back for this